inc/meta-box-models.php

```diff
@@ -338,7 +338,7 @@ public function validate( $post_ID, $field ) {
     // $data = array_intersect_key($_POST, $this->fields);
     if ( array_key_exists( 'meta_key', $field ) ) {
         $key = $field['meta_key'];
-    } elseif ( array_key_exists( $field['taxonomy'], $field ) ) {
+    } elseif ( array_key_exists( 'taxonomy', $field ) ) {
```

Comment:
@gboone This was the line that ended up being the main troublemaker for us when we were emailing back and forth last Friday.